src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -21,23 +21,54 @@ object Lorre extends App with LazyLogging {
   lazy val db = DatabaseUtil.db
   val tezosNodeOperator = new TezosNodeOperator(TezosNodeInterface)
 
-  try {
-    while(true) {
-      logger.info("Fetching blocks")
-      processTezosBlocks()
-      logger.info("Fetching accounts")
-      processTezosAccounts()
-      logger.info("Taking a nap")
-      Thread.sleep(sleepIntervalInSeconds * 1000)
-    }
-  } finally db.close()
+  val usage =
+    """
+      Usage: runMain tech.cryptonomic.conseil.Lorre --platform <platform> --network <network>
+    """.stripMargin
+
+  val argsList = args.toList
+  val (platform, network) = argsList match {
```

Comment:
What was the conclusion here? Were we going to use a proper arg parsing library or just assume a pattern for now?

Reply:
I thought we were just doing this for now, since we're the ones calling it from the command line mostly.

Reply:
Can you open a ticket then for using an arg parsing library?

Reply:
I opened https://github.com/Cryptonomic/Conseil/issues/121.